.eslintrc.json

```diff
@@ -10,6 +10,8 @@
     {
       "files": ["**/*.ts?(x)"],
       "plugins": ["@typescript-eslint", "no-relative-import-paths"],
+      "extends": ["plugin:@typescript-eslint/recommended"],
+      "parser": "@typescript-eslint/parser",
       "rules": {
         "@typescript-eslint/consistent-type-imports": "error",
         "@typescript-eslint/no-unused-vars": "error",
```

Comment:
as far as I can tell, this rule is included within @typescript-eslint/recommend and should not need to be redeclared here

https://typescript-eslint.io/rules/no-unused-vars

does the lint result change without this line?

```suggestion
```

Reply:
No, lint results does not change with this change.
Committing this change, thank you for the sugestion.

Reply:
I committed the change, it did not result  in any error on local machine but it resulted in some errors in pre-checks. Thur reverted back the change.